src/fabric_cli/parsers/fab_fs_parser.py

```diff
@@ -418,23 +428,27 @@ def register_set_parser(subparsers: _SubParsersAction) -> None:
         "# assign a custom pool",
         "$ set ws2r.workspace -q sparkSettings.pool.defaultPool -i <inline_json_w_id_name_type> -f",
     ]
+    set_learnmore = [
+        "Tip: Run `set <path> -q check -i empty` to see updatable  properties (e.g. set ws1.Workspace -q check -i empty)"
```

Comment:
why did you add this? is it really working? I removed it in my PR, maybe a merge went wrong?